README.md

```diff
@@ -40,3 +40,81 @@ if ($file->getSize()) {
 // Access a resource provided by this Divide.IQ server.
 $result = $divideIq->request('stockbase_stock');
 ```
+
+Example Usage orderRequest POST
+=============
+```php
+use DivideBV\PHPDivideIQ\DivideIQ;
+use DivideBV\PHPDivideIQ\Models\OrderDelivery;
+use DivideBV\PHPDivideIQ\Models\OrderHeader;
+use DivideBV\PHPDivideIQ\Models\Orderline;
+use DivideBV\PHPDivideIQ\Models\OrderRequest;
+use DivideBV\PHPDivideIQ\Models\Person;
+use DivideBV\PHPDivideIQ\Models\Address;
```

Comment:
Agree, fixed it in this commit: https://github.com/DivideBV/PHPDivideIQ/pull/1/commits/61cf0f3cf4c14fc5ec39e20fc3fbdc1d4dad6e82